app/models/feedback.rb

```diff
@@ -1,4 +1,6 @@
 class Feedback < ApplicationRecord
+  enum service: {check: :check, aytq: :aytq}
```

Comment:
The enum values need to be strings rather than symbols, otherwise they don't get set properly on the feedback ActiveRecord object.